builder/scaleway/builder_acc_test.go

```diff
@@ -39,7 +39,7 @@ func testAccPreCheck(t *testing.T) bool {
 
 const testBuilderAccBasic = `
 source "scaleway" "basic" {
-  commercial_type      = "DEV1-S"
+  commercial_type      = "PRO2-XXS"
```

Comment:
What is the justification for this change?

Reply:
Speeds up the test twice as fast